Classes/Resource/BynderDriver.php

```diff
@@ -506,13 +501,13 @@ public function getPermissions($identifier)
      * buffer before. Will be taken care of by the Storage.
      *
      * @param  string  $identifier
-     * @throws Exception\FileDoesNotExistException
+     * @throws \TYPO3\CMS\Core\Resource\Exception\FileDoesNotExistException
      */
     public function dumpFileContents($identifier)
     {
         try {
             $downloadLocation = $this->getAssetBankManager()->getMediaDownloadLocation($identifier)->wait();
-            readfile($downloadLocation['s3_file'], 0);
+            readfile($downloadLocation['s3_file'], false);
```

Comment:
You can speed up everything if you add the global namespace to the native php functions.
```suggestion
            \readfile($downloadLocation['s3_file'], false);
```